packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -72,6 +78,18 @@ export function Comments( {
 		);
 	}, [] );
 
+	const [ blocks ] = useEntityBlockEditor( 'postType', postType, {
+		id: postId,
+	} );
```

Comment:
What's the reason for switching to `useEntityBlockEditor`? This reverts the suggestion from https://github.com/WordPress/gutenberg/pull/66873#discussion_r1869047714.

Reply:
@Mamaduka - used `useEntityBlockEditor` for the functionality to work in Site Editor. Before it was not.